django/views/templates/default_urlconf.html

```diff
@@ -79,137 +67,51 @@
           }
           @keyframes smoke {
             0% {
-              -webkit-transform: scale(1.58, 1.2) translateY(-55px) skew(3deg);
-              transform: scale(1.58, 1.2) translateY(-55px) skew(3deg);
-            }
-            100% {
-              -webkit-transform: scale(1.6, 1.22) translateY(-55px) skew(-3deg);
-              transform: scale(1.6, 1.22) translateY(-55px) skew(-3deg);
-            }
-          }
-          @-webkit-keyframes smokeFirefox {
-            0% {
-              -webkit-transform: scale(1.58, 1.2) translateY(-75px) skew(0);
-              transform: scale(1.58, 1.2) translateY(-75px) skew(0);
+              transform: translate3d(-5px, 0, 0);
             }
             100% {
-              -webkit-transform: scale(1.58, 1.21) translateY(-75px) skew(1deg);
-              transform: scale(1.58, 1.21) translateY(-75px) skew(1deg);
+              transform: translate3d(5px, 0, 0);
             }
           }
-          @keyframes smokeFirefox {
-            0% {
-              -webkit-transform: scale(1.58, 1.2) translateY(-75px) skew(0);
-              transform: scale(1.58, 1.2) translateY(-75px) skew(0);
-            }
-            100% {
-              -webkit-transform: scale(1.58, 1.21) translateY(-75px) skew(1deg);
-              transform: scale(1.58, 1.21) translateY(-75px) skew(1deg);
-            }
-          }
-          .flame 0%, .flame 100% {
-            -webkit-transform: translate(265px, 249px) scale(1.9) rotate(180deg);
-            transform: translate(265px, 249px) scale(1.9) rotate(180deg);
-          }
           .flame {
-            -webkit-animation: burnInner2 .1s infinite ease-in-out alternate;
-            animation: burnInner2 .1s infinite ease-in-out alternate;
-          }
-          @-webkit-keyframes burnInner2 {
-            0% {
-              -webkit-transform: translate(265px, 249px) scale(1.9) rotate(180deg) skew(5deg);
-              transform: translate(265px, 249px) scale(1.9) rotate(180deg) skew(5deg);
-            }
-            100% {
-              -webkit-transform: translate(265px, 253px) scale(2.2) rotate(180deg) skew(-5deg);
-              transform: translate(265px, 253px) scale(2.2) rotate(180deg) skew(-5deg);
-            }
+            animation: burnInner2 .1s 70 ease-in-out alternate;
           }
           @keyframes burnInner2 {
             0% {
-              -webkit-transform: translate(265px, 249px) scale(1.9) rotate(180deg) skew(5deg);
-              transform: translate(265px, 249px) scale(1.9) rotate(180deg) skew(5deg);
+              transform: translate3d(0, 0, 0);
             }
             100% {
-              -webkit-transform: translate(265px, 253px) scale(2.2) rotate(180deg) skew(-5deg);
-              transform: translate(265px, 253px) scale(2.2) rotate(180deg) skew(-5deg);
+              transform: translate3d(0, 3px, 0);
             }
           }
-          @-webkit-keyframes thurst {
+          @keyframes thurst {
```

Comment:
We could fix the typo while you're here: `thurst` → `thrust`.

Reply:
Wow! I have been looking at this code for hours and didn’t even notice. Thank you, will update later today :)